### ISSUE TITLE
Update `arm64` image

### ISSUE THREAD

**Issue by vnghia** [juicedata/docker-volume-juicefs#35]
Hello, the `arm64` image for this plugin is quite old at this point (~2 years ago). Could you update it ? Thank you very much !

**Comment by powerpaul17** [comment]
Yes, please. An arm64 version would be much appreciated. Is there something to help with?

**Comment by powerpaul17** [comment]
I created a pull request which should allow building this for arm64: #38
